packages/shadow-dom/src/createShadowDOMRenderer.ts

```diff
@@ -14,6 +14,13 @@ const SUPPORTS_CONSTRUCTABLE_STYLESHEETS: boolean = (() => {
   }
 })();
 
+export interface CreateShadowDomRendererOptions {
+  /**
+   * If specified, a renderer will insert created CSSStyleSheets after this CSSStyleSheet.
+   */
+  insertionPoint?: ExtendedCSSStyleSheet;
```

Comment:
Why does this need to be an `ExtendedCSSStyleSheet` should a normal `CSSStylesheet ` be enough?

Reply:
It doesn't need to be `ExtendedCSSStyleSheet` but the shadow DOM renderer already [exposes this as the type of its `adoptedStyleSheets`](https://github.com/microsoft/griffel/blob/main/packages/shadow-dom/src/types.ts#L8) and [`findInsertionPoint()` uses this type](https://github.com/microsoft/griffel/blob/main/packages/shadow-dom/src/findInsertionPoint.ts#L11) so using it here feels more consistent to me.

Of course, currently we don't use any of the extended parts so we don't need to use this type but I think the API consistency is nice. That said, don't feel strongly about this.

Reply:
Done!